deploy/examples/common.yaml

```diff
@@ -896,6 +896,7 @@ rules:
       - create
       - update
       - delete
+      - deletecollection
```

Comment:
I see why so many tests are failing now. This needs to be modified in the helm chart and then run `make gen-rbac`

Reply:
@BlaineEXE Done